client/packages/invoices/src/Prescriptions/DetailView/PrescriptionLineEdit/PageLayout.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Grid } from '@openmsupply-client/common';
+
+interface PageLayoutProps {
+  Left: React.ReactElement;
+  Right: React.ReactElement;
+}
+
+export const PageLayout = ({ Left, Right }: PageLayoutProps) => {
+  return (
+    <Grid container spacing={2} direction="row" padding={2} paddingBottom={2}>
+      <Grid item xs={3}>
+        {Left}
+      </Grid>
+      <Grid item xs={9}>
```

Comment:
This is a different ratio to Manual reqs (seems like they aren't using all `12` available slots for some reason?)

Reply:
Should this be some kind of common component?

Reply:
https://github.com/msupply-foundation/open-msupply/issues/5766